gradle.properties

```diff
@@ -1,4 +1,4 @@
-modName = Better Loading Screen GT:NH
+modName = Better Loading Screen GTNH
```

Comment:
Because we seem to use GTNH much _much_ more than GT:NH:

```
bombcar@lappy386 ~/Documents/GitHub $ grep GTNH * -R | wc -l
    5257
bombcar@lappy386 ~/Documents/GitHub $ grep GT\:NH * -R | wc -l
      24
```